Bluetooth HIDS samples: describe known issue for unexpected disconnections due to the LL collisions (#22775)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * Fixed an issue where the sample would assert with the :kconfig:option:`CONFIG_ASSERT` Kconfig option enabled.
    This was due to calling the :c:func:`bt_iso_chan_send` function from a timer ISR handler and sending SDUs to the controller with invalid timestamps.

* :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples:

>   * Fixed an issue where the samples would disconnect due to Bluetooth Link Layer procedure collisions.

Reviewer: Discussed offline

>   * Worked around an issue with unexpected disconnections that resulted from improper handling of the Bluetooth Link Layer procedures by the connected Bluetooth Central device.

Reviewer 2: ```suggestion
  * Added a workaround to an issue with unexpected disconnections that resulted from improper handling of the Bluetooth Link Layer procedures by the connected Bluetooth Central device.
```
